shiny/playwright/controller/_accordion.py

```diff
@@ -313,6 +317,55 @@ def set(
                 )
             self.accordion_panel(elem_value).set(elem_value in open, timeout=timeout)
 
+    def expect_class(
+        self,
+        class_name: str,
+        *,
+        has_class: bool = True,
```

Comment:
I'm going to remove this kwarg. If they're looking to find the class that they've put in, this method makes sense. 

I do not believe it makes sense to expect on a missing class value.

```suggestion
```